server/services/nl.py

```diff
@@ -17,8 +17,13 @@
 from sentence_transformers import SentenceTransformer
 from sentence_transformers.util import semantic_search
 
+from lib.nl_detection import NLClassifier, ClassificationType
+from lib.nl_detection import CorrelationClassificationAttributes
+from lib.nl_detection import ContainedInClassificationAttributes, ContainedInPlaceType
+from lib.nl_detection import RankingClassificationAttributes, RankingType
+from lib.nl_detection import PeriodType, TemporalClassificationAttributes
```

Comment:
from lib.nl_detection import a,b,c,d,...

Reply:
Happy to do it hat way but that means the one import line is extremely long and very hard to use. So I kept them logically grouped. Let me know if you feel this must be changed and I will modify in the next commit :) Thanks for the late night review!

Reply:
This is definitely optional. Good to go!